### ISSUE TITLE
Saving and loading the "View"

### ISSUE THREAD

**Issue by ilka2** [CG-Tespy/Fungus-Slot-based-Save-System#21]
I have a problem implementing save/load. The text is saved, but the background (View component) is not loaded. When I load a save, I just have a blue background instead of my image. The "Say Dialog" character image is also not saved. If the save system does not support saving images, can you tell me what to do and which scripts to use? If I can still implement the icons of the characters myself, the choice of "View" is made through Flowchart, which I am far from.

Basically, the "view" is the camera position. If I understand how the camera interacts with the "view", I will be able to save and load the background on my own, bypassing Fungus.

Happy New Year

**Comment by CG-Tespy** [comment]
Sorry for responding so late. As for the backgrounds, character images, and camera positions not being saved...
Yes, the system does not save that stuff. As for how to get around that, here are a couple solutions:

1. Set up Boolean Variables in Fungus, updating them as appropriate. Then in Save Data Loaded blocks, check those variables to see which backgrounds, character images, and camera positions should be loaded.
2. Create your own DataSaver and SaveLoader scripts that save and load that stuff for you. If you are confident in your C# skills, this is probably the better option

**Comment by ilka2** [comment]
Hello. Fungus is giving me a lot of trouble and wasting my time. Since I have many different ideas and my own vision, I hardly rely on internal block diagrams, but write my own scripts. Because of this, there are constant conflicts with the "security system" of Fungus, which protects me from myself. And I thought that saving the last image would be as difficult as everything else. But after figuring out the issue, I realized that the camera moves along the "View" and their coordinates fully correspond to each other. I hope it will be enough to save the camera's coordinates and set them to the camera in the scene when loading. Haven't tried it yet, but it sounds logical and not complicated.
